src/hooks/useBreakpoints.ts

```diff
@@ -2,7 +2,7 @@ import { useEffect, useState } from 'react'
 
 const xsSize = 279
 const smSize = 374
-const mobileSize = 399
+const mobileSize = 420
```

Comment:
![image](https://user-images.githubusercontent.com/49658988/172862237-dd1a09b9-c0a0-45d7-8101-546ef81419fa.png)
Iphone 11 Pro Max has 414 width, so I decided to extend this